CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - connection_pool renamed to pool (#239)
 - Use msgpack/v5 instead of msgpack.v2 (#236)
 - Call/NewCallRequest = Call17/NewCall17Request (#235)
+- Change encoding of the queue.Identify() UUID argument from binary blob to
+  plain string. Needed for upgrade to Tarantool 3.0, where a binary blob is
+  decoded to a varbinary object (tarantool/tarantool#1629).
```

Comment:
Please link to the current pull request to keep the changelog in the same style.
```suggestion
  decoded to a varbinary object (#313).
```

Reply:
Done.